astropy/io/fits/tests/test_core.py

```diff
@@ -1264,6 +1264,16 @@ def _make_gzip_file(self, filename='test0.fits.gz'):
 
         return gzfile
 
+    def test_write_overwrite(self):
+        filename = self.temp('test_overwrite.fits')
+        hdu = fits.PrimaryHDU(data=np.arange(10))
+        hdu.writeto(filename)
+        msg = (r'File .* already exists. If you mean to replace it then use '
+               'the argument "overwrite=True".')
```

Comment:
```suggestion
        msg = (r'File .* already exists\. If you mean to replace it then use '
               'the argument "overwrite=True"\.')
```

Reply:
I noticed I have only added the slashes to the very last period. I'll fix this in the next commit.